rosidl_typesupport_opendds_c/resource/msg__type_support_c.cpp.em

```diff
@@ -199,7 +196,7 @@ if isinstance(type_, AbstractNestedType):
     size_t size = @(member.type.size);
 @[    else]@
     size_t size = ros_message->@(member.name).size;
-    if (size > (std::numeric_limits<DDS_Long>::max)()) {
+    if (size > (std::numeric_limits<int>::max)()) {
```

Comment:
I thought there was a CORBA type for this? This doesn't seem right to me to assume the max size for a DDS sequence is (and will always be) int. This also can change if compiled on different architectures or using different compilers.

Reply:
Since this is an .em template, it's hard to see where it really gets used.  I don't mind int here, at least to get started.  Is someone going to have a collection of more than 2 billion elements?